src/utils/index.js

```diff
@@ -1,46 +1,47 @@
 /* global location */
 
 /* Centralized place to reference utilities since utils is exposed to the user. */
-var debug = require('./debug');
-var deepAssign = require('deep-assign');
-var device = require('./device');
-var objectPool = require('./object-pool');
+import debug from './debug.js';
+import deepAssign from 'deep-assign';
+import * as device from './device.js';
+import * as objectPool from './object-pool.js';
 
 var warn = debug('utils:warn');
 
 /** @deprecated */
-module.exports.bind = function (fn) {
+export function bind (fn) {
   return fn.bind.apply(fn, Array.prototype.slice.call(arguments, 1));
-};
-module.exports.coordinates = require('./coordinates');
-module.exports.debug = debug;
-module.exports.device = device;
-module.exports.entity = require('./entity');
-module.exports.forceCanvasResizeSafariMobile = require('./forceCanvasResizeSafariMobile');
-module.exports.material = require('./material');
-module.exports.objectPool = objectPool;
-module.exports.split = require('./split').split;
-module.exports.styleParser = require('./styleParser');
-module.exports.trackedControls = require('./tracked-controls');
-
-module.exports.checkHeadsetConnected = function () {
+}
+export * as coordinates from './coordinates.js';
+export { default as debug } from './debug.js';
+export * as device from './device.js';
+export * as entity from './entity.js';
+export { default as forceCanvasResizeSafariMobile } from './forceCanvasResizeSafariMobile.js';
+export * as material from './material.js';
+export * as objectPool from './object-pool.js';
+export { split } from './split.js';
+export * as styleParser from './styleParser.js';
+export * as trackedControls from './tracked-controls.js';
+
+export function checkHeadsetConnected () {
   warn('`utils.checkHeadsetConnected` has moved to `utils.device.checkHeadsetConnected`');
   return device.checkHeadsetConnected(arguments);
-};
-module.exports.isGearVR = module.exports.device.isGearVR = function () {
+}
+export function isGearVR () {
   warn('`utils.isGearVR` has been deprecated, use `utils.device.isMobileVR`');
-};
-module.exports.isIOS = function () {
+}
+
+export function isIOS () {
   warn('`utils.isIOS` has moved to `utils.device.isIOS`');
   return device.isIOS(arguments);
-};
-module.exports.isOculusGo = module.exports.device.isOculusGo = function () {
+}
+export function isOculusGo () {
   warn('`utils.isOculusGo` has been deprecated, use `utils.device.isMobileVR`');
-};
-module.exports.isMobile = function () {
+}
```

Comment:
So before isGearVR and isOculusGo were available on AFRAME.utils and AFRAME.utils.device I see.
Those functions are deprecated since a long time now.
Shouldn't we just remove those two functions now @dmarcos? We also removed gearvr-controls from the repo.

Reply:
yeah. we can remove those functions

Reply:
done in #5635